### DISCUSSION TITLE
大佬，软件会导致无法开启睿频

### DISCUSSION THREAD

**Discussion by dajidali918** [zqhong/RyzenTuner#20] [General]
用这个软件之后，电源计划里面设置了cpu电源状态，设置为百分百才能开睿频，但win10好像不能设置。。。

**Comment by genji7430** [comment]
大佬你解决了吗win10那个设置了百分之百还是锁在了3ghz，之前正常都能跑到4ghz的，重装了bios删除了这个软件还是没用恢复

**Comment by dajidali918** [comment]
解决了


win11掉睿频的电源计划里面设置cpu状态为百分百，win10下载Lenovo Quick Fix：电脑高性能管理工具就OK了
https://iknow.lenovo.com.cn/detail/dc_178676.html ，Lenovo Quick Fix：电脑高性能管理工具


有睿频后不要用ryzentuner的性能模式就行了，别的模式不影响睿频



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月13日(周一) 凌晨0:01
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [zqhong/RyzenTuner] 大佬，软件会导致无法开启睿频 (Discussion #20)




 
大佬你解决了吗win10那个设置了百分之百还是锁在了3ghz，之前正常都能跑到4ghz的，重装了bios删除了这个软件还是没用恢复

**Reply by genji7430** [reply]
感谢大佬确实解决了，win10上有处理器电源状态但我手动设置也没用，另外请问用amd官网的ryzen master可以将cpu频率固定在一个值吗或者有其他可以实现这个功能的软件推荐吗，感谢

**Comment by dajidali918** [comment]
百度一下有其他软件，但我只用过这一个，因为别的软件主要是为了超频，有的也不支持r7 6800h。
这个软件挺好用的，别再点性能模式就行了，即使点了也没事，再用Lenovo Quick Fix开一下就恢复睿频了。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月13日(周一) 凌晨0:30
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [zqhong/RyzenTuner] 大佬，软件会导致无法开启睿频 (Discussion #20)




 
感谢大佬确实解决了，win10上有处理器电源状态但我手动设置也没用，另外请问用amd官网的ryzen master可以将cpu频率固定在一个值吗或者有其他可以实现这个功能的软件推荐吗，感谢

**Reply by genji7430** [reply]
我刚刚试了只要点了手动模式，他会自动切到性能模式然后又锁频，这应该是个bug

**Reply by genji7430** [reply]
进一步试验了，初步判断是功率的问题，手动设置26w也会和性能模式的26w一样降频，手动设置25和27,28均能实现睿频。

**Comment by dajidali918** [comment]
我手动设置的120w，没有锁频



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月13日(周一) 凌晨0:45
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [zqhong/RyzenTuner] 大佬，软件会导致无法开启睿频 (Discussion #20)




 
我刚刚试了只要点了手动模式，他会自动切到性能模式然后又锁频，这应该是个bug

**Comment by dajidali918** [comment]
OK，问题解决，这软件还是挺好用的，我经常用这个限制功率。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月13日(周一) 凌晨1:22
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [zqhong/RyzenTuner] 大佬，软件会导致无法开启睿频 (Discussion #20)




 
进一步试验了，初步判断是功率的问题，手动设置26w也会和性能模式的26w一样降频，手动设置25和27,28均能实现睿频。

**Comment by zqhong** [comment]
我了解看看可能的原因，谢谢反馈

**Comment by zqhong** [comment]
新版本增加了开启/关闭睿频的开关

**Comment by dajidali918** [comment]
好的，谢谢大佬！



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2026年7月16日(周四) 中午11:25
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [zqhong/RyzenTuner] 大佬，软件会导致无法开启睿频 (Discussion #20)




 
新版本增加了开启/关闭睿频的开关
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today!

**Comment by dajidali918** [comment]
感谢大佬更新，我下载更新一下



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2026年7月16日(周四) 中午11:25
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [zqhong/RyzenTuner] 大佬，软件会导致无法开启睿频 (Discussion #20)




 
新版本增加了开启/关闭睿频的开关
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today!
